recipe-server/client/control_new/components/common/CurrentUser.js

```diff
@@ -0,0 +1,38 @@
+import { Button } from 'antd';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { getCurrentUser } from 'control_new/state/serviceInfo/selectors';
+
+
+@connect(
+  state => ({
+    user: getCurrentUser(state),
+  }),
+)
+export default class CurrentUser extends React.Component {
```

Comment:
This name feels a little ambiguous, but I can't think of anything better. `CurrentUserPanel` or something? I was expecting this to be a query component when I first saw the name.